### PR TITLE
[gui-test][full-ci] update UI object selectors

### PR DESCRIPTION
**Pull request by saw-jan**
Fixed the object selectors according to the new changes.

NOTE: scenario to show log dialog with ctrl+l has been removed
>Removed on purpose.
The dialog is accessible through the settings tab.

Closes https://github.com/owncloud/client/issues/11634

### CODE REVIEW

File: test/gui/tst_checkAlltabs/test.feature [hunk 1]
```diff
@@ -8,13 +8,13 @@ Feature: Visually check all tabs
...
-            | QuitOwncloud |
-
+        Then the toolbar should have the following tabs:
+            | Add Account |
+            | Activity    |
+            | Settings    |
+            | Quit        |
 
+    # Q: log dialog is not opened
     Scenario: Open log dialog with Ctrl+l keys combination
```
Comment (saw-jan) [inline test/gui/tst_checkAlltabs/test.feature:18]: `ctrl+l` doesn't open the log dialog. was it changed or regression?
CC @TheOneRing

File: test/gui/shared/steps/account_context.py [hunk 2]
```diff
@@ -163,6 +167,8 @@ def step(context, username, host):
     displayname = substituteInLineCodes(displayname)
     host = substituteInLineCodes(host)
 
+    # Q: recently added account is not focused
+    Toolbar.openAccount(displayname, host)
```
Comment (saw-jan) [inline test/gui/shared/steps/account_context.py:171]: Recently added account doesn't get the focus. The view is still in the old account. Is it expected behavior now?

CC @TheOneRing

File: test/gui/tst_checkAlltabs/test.feature (hunk 1 above)
Comment (TheOneRing) [reply test/gui/tst_checkAlltabs/test.feature:18]: Removed on purpose.
The dialog is accessible through the settings tab.
I'll open an issue to update doc.

File: test/gui/shared/steps/account_context.py (hunk 2 above)
Comment (TheOneRing) [reply test/gui/shared/steps/account_context.py:171]: Thx for informing me about the regression could you please open an issue?

Comment (saw-jan) [reply test/gui/shared/steps/account_context.py:171]: here is the issue: https://github.com/owncloud/client/issues/11642

**Comment by saw-jan** [comment]
CI fails due to existing known issue: https://github.com/owncloud/client/issues/11619

File: test/gui/tst_syncing/test.feature [hunk 3]
```diff
@@ -31,7 +31,7 @@ Feature: Syncing files
         And the folder "simple-folder" should exist on the file system
         And the folder "large-folder" should exist on the file system
 
-    @issue-9733
+    @issue-9733 @skip
```
Comment (saw-jan) [inline test/gui/tst_syncing/test.feature:34]: I will work on separate PR for these skipped tests which are causing a bit of flakiness in the CI

Comment (saw-jan) [reply test/gui/tst_syncing/test.feature:34]: issue https://github.com/owncloud/client/issues/11619 occurs more in these scenarios

**Review by Salipa-Gurung — APPROVED** [review]

**Pull request merged**